_inc/client/plans/plan-body.jsx

```diff
@@ -50,126 +50,149 @@ const PlanBody = React.createClass( {
-                      this.props.isPluginInstalled( 'vaultpress/vaultpress.php' )
-                      && this.props.isPluginActive( 'vaultpress/vaultpress.php' ) ? (
-                        <Button href="https://dashboard.vaultpress.com/" className="is-primary">
-                          { __( 'View your security dashboard' ) }
-                        </Button>
-                      )
-                      : (
-                        <Button href={ 'https://wordpress.com/plugins/setup/' + this.props.siteRawUrl + '?only=vaultpress' } className="is-primary">
-                          { __( 'Configure VaultPress' ) }
-                        </Button>
-                      )
-                  }
-                </div>
-                : ''
-            }
+					{
+						includes( [ 'jetpack_business', 'jetpack_business_monthly' ], this.props.plan ) ?
+							<div className="jp-landing__plan-features-card">
+								<h3 className="jp-landing__plan-features-title">{ __( 'Backups & Security Scanning' ) }</h3>
+								<p>{ __( 'Real-time backup of all your site data with unlimited space, one-click restores, automated security scanning, one-click threat resolution, and priority support (powered by VaultPress). Also includes surveys, polls and ratings.' ) }</p>
+								{
+									this.props.isFetchingPluginsData ? '' :
+									this.props.isPluginInstalled( 'vaultpress/vaultpress.php' )
+									&& this.props.isPluginActive( 'vaultpress/vaultpress.php' ) ? (
+										<Button href="https://dashboard.vaultpress.com/" className="is-primary">
+											{ __( 'View your security dashboard' ) }
+										</Button>
+									)
+									: (
+										<Button href={ 'https://wordpress.com/plugins/setup/' + this.props.siteRawUrl + '?only=vaultpress' } className="is-primary">
+											{ __( 'Configure VaultPress' ) }
+										</Button>
+									)
+								}
+							</div>
+						: ''
+					}
 
-						{
-							includes( [ 'jetpack_business', 'jetpack_business_monthly' ], this.props.plan ) ?
-								<div className="jp-landing__plan-features-card">
-									<h3 className="jp-landing__plan-features-title">{ __( 'Surveys & Polls' ) }</h3>
-									<p>{ __( 'Unlimited surveys, unlimited responses. Use the survey editor to create surveys quickly and easily. Collect responses via your website, e-mail or on your iPad or iPhone.' ) }</p>
-									<Button href="https://polldaddy.com/dashboard/" className="is-primary">
-										{ __( 'Create a new poll' ) }
-									</Button>
-								</div>
-								: ''
-						}
+					{
+						includes( [ 'jetpack_premium', 'jetpack_premium_monthly', 'jetpack_business', 'jetpack_business_monthly' ], this.props.plan ) ?
+							<div className="jp-landing__plan-features-card">
+								<h3 className="jp-landing__plan-features-title">{ __( 'Video Hosting' ) }</h3>
+								<p>{ __( 'Fast, optimised, ad-free, and unlimited video hosting for your site (powered by VideoPress).' ) }</p>
```

Comment:
Do we offer unlimited video hosting to Premium users as well? On WordPress.com, space is limited to 13GB for Premium users; only Business users get unlimited space.